plugin/builtin/patchtransformer/PatchTransformer_test.go

```diff
@@ -692,9 +691,6 @@ spec:
 `)
 }
 
-// test for https://github.com/kubernetes-sigs/kustomize/issues/2767
-// currently documents broken state.  resulting ports: should have both
-// take-over-the-world and disappearing-act on 8080
 func TestPatchTransformerSimilarArrays(t *testing.T) {
```

Comment:
Lets keep a comment referencing #2767 and add a pointer to 
https://github.com/kubernetes/kubernetes/issues/39188
We should see how or if we can resolve this upstream.
Also add a comment pointing to the "openapi" target in the Makefile.